File: app/src/main/java/com/vondear/tools/model/CityModel.java
package com.vondear.tools.model;

import com.vondear.tools.tools.RxPullXml;

import java.util.List;

/**
 * Created by Android Studio
 * user : Vondear
 * date : 2017/3/14 ${time}
 * desc :
 */
public class CityModel {

    private String cityName;
    private String cityCode;
    private RxPullXml.unitType cityType;
    private List<CityModel> countyList;

    public String getCityName() {
        return cityName;
    }

    public void setCityName(String cityName) {
        this.cityName = cityName;
    }

    public String getCityCode() {
        return cityCode;
    }

    public void setCityCode(String cityCode) {
        this.cityCode = cityCode;
    }

    public RxPullXml.unitType getCityType() {
        return cityType;
    }

    public void setCityType(RxPullXml.unitType cityType) {
        this.cityType = cityType;
    }

    public List<CityModel> getCountyList() {
        return countyList;
    }

    public void setCountyList(List<CityModel> countyList) {
        this.countyList = countyList;
    }
}

File: RxDemo/src/main/java/com/tamsiree/rxdemo/model/CityModel.kt
package com.tamsiree.rxdemo.model

import com.tamsiree.rxdemo.tools.RxPullXml.unitType

/**
 * Created by Android Studio
 * user : Tamsiree
 * date : 2017/3/14 ${time}
 * desc :
 */
class CityModel {

    var cityName: String? = null
    var cityCode: String? = null
    var cityType: unitType? = null
    var countyList: MutableList<CityModel>? = null


    constructor()

    constructor(cityName: String?, cityCode: String?, cityType: unitType?, countyList: MutableList<CityModel>?) {
        this.cityName = cityName
        this.cityCode = cityCode
        this.cityType = cityType
        this.countyList = countyList
    }


}